actions/notify-slack/get-info.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+function get-info() {
+  set -eo pipefail
+
+  if [ -z "${EMOJI:-}" ]; then
+
+    # MESSAGE="A $TYPE for $APPLICATION:$VERSION to $ENVIRONMENT resulted in $STATUS"
+    case "${STATUS:-}" in
+    success)
+      EMOJI=large_green_circle
+      ;;
+    failure)
+      EMOJI=red_circle
+      ;;
+    started)
+      EMOJI=large_blue_circle
+      # MESSAGE="A $TYPE has been started for $APPLICATION:$VERSION to $ENVIRONMENT"
+      ;;
+    *)
+      EMOJI=white_circle
+      ;;
+    esac
+
+  fi
+  echo emoji="$EMOJI" >>"$GITHUB_OUTPUT"
+  TIMESTAMP="$(date +%s)"
+  MESSAGE="A ${TYPE:-}for ${APPLICATION:-}:${VERSION:-} to ${ENVIRONMENT:-} resulted in ${STATUS:-}"
```

Comment:
We should not be defaulting these to empty string if they are required such as TYPE, APPLICATION, VERSION, etc.  When running the script locally you should just set these variables in your shell.

```suggestion
  MESSAGE="A ${TYPE} for ${APPLICATION}:${VERSION} to ${ENVIRONMENT} resulted in ${STATUS}"
```